lms/djangoapps/ccx/models.py

```diff
@@ -32,6 +32,8 @@ class CustomCourseForEdX(models.Model):
     # if not empty, this field contains a json serialized list of
     # the master course modules
     structure_json = models.TextField(verbose_name='Structure JSON', blank=True, null=True)
+    # Custom Json field to add any additional information to the CCX model
+    custom_content = models.JSONField(default=dict, blank=True, null=True, verbose_name="Custom Json Content")
```

Comment:
I think we should name it the same as "other course settings", as they both have the same purpose and having the same name could easily guide the people who will work with this. What do you think? @JuanDavidBuitrago

Reply:
@Squirrel18 I think it's a good idea, and it makes a lot of sense. Variable name changed.